examples/amp-story/bookendv1.json

```diff
@@ -15,19 +15,19 @@
       "type": "small",
       "title": "This is an example article",
       "url": "http://example.com/article.html",
-      "image": "http://placehold.it/256x128"
+      "image": "http://placehold.it/100x100"
```

Comment:
I liked the idea of having images that don't fit their container, it makes it easier to notice any cropping issue.